### ISSUE TITLE
failed chat message event handling

### ISSUE THREAD

**Issue by nuvasuper** [rauchg/chat-example#26]
In your example on http://socket.io/get-started/chat/, in index.html, the code for binding the event `http://socket.io/get-started/chat/` etc is in the lines BEFORE the form is declared `<form action="">
      <input id="m" autocomplete="off" /><button>Send</button>
    </form>`  Which causes it to fail unless reordered!  The <script> section for that code ought to go after the form has been created.

**Comment by Almenon** [comment]
Ran into same problem.  Either the code should be activated upon document ready or it should be moved below body.

**Comment by halink0803** [comment]
@rauchg can you change the tutorial following this then newbie will not be confused why socket.io not working.

**Comment by dsprogis** [comment]
Yeah, this issue had me scratching my head for a while!  By moving the script section after the body, it started working.  Rather than reading the FAQ's I googled the problem and there are a bunch of posts on StackOverflow about this problem. 

There are also a few about res.sendfile (deprecated).  It would be nice to see both cleaned up in the tutorial.

**Comment by skipgamer** [comment]
Just adding my voice to the chorus that this should be updated. Thanks for the issue and fix https://github.com/rauchg/chat-example/pull/13
